src/bytewise.rs

```diff
@@ -994,4 +977,29 @@ mod tests {
         assert_eq!(0x80, pma.states[0].base().unwrap().get());
         assert_eq!(0x100, pma.states[0x80].base().unwrap().get());
     }
+
+    #[test]
+    fn test_serialize_state() {
+        let mut opos_ch = U24nU8::default();
+        opos_ch.set_a(U24::try_from(57).unwrap());
+        opos_ch.set_b(77);
+        crate::serializer::tests::test_common(State {
```

Comment:
Could you not hide the test code inside the function?

In unit testing, it is often said DAMP not DRY.
It is not clear what the behavior of `test_common` is, so we need to move on to the definition of this function.
https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests

Reply:
I removed `test_common` https://github.com/daac-tools/daachorse/pull/68/commits/32618fa25af522d9bb9aaece0f6923a135e0352f
I learned a lot, thanks!